pkg/localize/locales/en/cmd/registry_crud.en.toml

```
@@ -130,6 +137,12 @@ one = 'Name of Service Registry instance:'
 [registry.cmd.create.input.name.help]
 one = 'Name can be any alphanumeric characters'
 
+[registry.cmd.create.input.description.message]
+one = 'Description of Service Registry instance [optional]:'
```

```suggestion
one = 'Description [optional]:'
```

Simply "Description" should be enough, like we do with Kafka. It should be clear from the context already.